estimagic/tests/conftest.py

```diff
@@ -1,7 +1,58 @@
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
 import pytest
 
 
 @pytest.fixture(scope="function", autouse=True)
 def fresh_directory(tmpdir):
     """Each test is executed in a fresh directory."""
     tmpdir.chdir()
```

Comment:
tmpdir is the old py.path thingy and will be deprecated. tmp_path returns a pathlib path
```suggestion
def fresh_directory(tmp_path):
    """Each test is executed in a fresh directory."""
    os.chdir(tmp_path)
```